src/runtime_src/core/include/xclperf.h

```diff
@@ -409,6 +409,7 @@ typedef struct {
   unsigned int TraceID;
   unsigned char Error;
   unsigned char Reserved;
+  bool isClockTrain = false;
```

Comment:
This doesn't look like "C" code.  I don't think in-class initialization is allowed in C.  xrt.h, which includes xclperf.h can be used in C programs.

Reply:
removed initialization